packages/v/vk-bootstrap/xmake.lua

```diff
@@ -0,0 +1,42 @@
+package("vk-bootstrap")
+    set_homepage("https://github.com/charles-lunarg/vk-bootstrap")
+    set_description("Vulkan Bootstrapping Iibrary.")
+    set_license("MIT")
+
+    add_urls("https://github.com/charles-lunarg/vk-bootstrap.git")
+
+    add_deps("vulkan-headers")
+
+    add_configs("shared", {description = "Build shared library.", default = false, type = "boolean", readonly = true})
+
+    if is_plat("linux") then
+        add_syslinks("dl")
+    end
+
+    on_install("windows", "linux", "macosx", function (package)
+        io.writefile("xmake.lua", [[
+            add_rules("mode.debug", "mode.release")
+            add_requires("vulkan-headers")
+            target("vk-bootstrap")
+                set_kind("static")
+                set_languages("cxx14")
+                add_files("src/VkBootstrap.cpp")
+                add_headerfiles("src/VkBootstrap.h")
+                add_headerfiles("src/VkBootstrapDispatch.h")
+                add_packages("vulkan-headers")
+                if is_plat("linux") then
+                    add_syslinks("dl")
+                end
+        ]])
+        import("package.tools.xmake").install(package, {"-m release"})
```

Comment:
only support release? you can remove `add_rules("mode.debug")` and only reserve mode.release

and remove `, {"-m release"}`

Reply:
It's a mistake. Done fixing.